src/index.html

```diff
@@ -28,45 +28,367 @@ <h3 id="logo">My Logo</h3>
         </label>
       </nav>
     </header>
+    <section class="bigger_section">
+      <section class="section">
+        <div class="container">
+          <div class="flex">
+            <h1 id="my_name">
+              Hey There.
+              <br />
+              I'm Zola.
+            </h1>
+            <h4 id="title">I'm a software Developer</h4>
+            <p id="describe_me">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hesitate to contact me.
+            </p>
+          </div>
+          <div class="flex social">
+            <ul class="socialIcons">
+              <li class="icons">
+                <a href="#"><img src="Img/GITHUB.png" alt="Github" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="Img/Linkedin.png" alt="LinkedIn" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"
+                  ><img src="Img/Angellist-Icon.png" alt="AngelList"
+                /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="Img/Twitter.png" alt="Twitter" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="Img/M.png" alt="Medium" /></a>
+              </li>
+            </ul>
+          </div>
+          <div id="design" class="flex"></div>
+        </div>
+      </section>
+
```

Comment:
Kindly I suggest you remove all blank unused blank lines in your project to make it more professional and to follow best practices. 😄